compat/src/index.d.ts

```diff
@@ -68,12 +68,858 @@ declare namespace React {
+		tableValues?: number | string | undefined;
+		targetX?: number | string | undefined;
+		targetY?: number | string | undefined;
+		textAnchor?: string | undefined;
+		textDecoration?: number | string | undefined;
+		textLength?: number | string | undefined;
+		textRendering?: number | string | undefined;
+		to?: number | string | undefined;
+		transform?: string | undefined;
+		u1?: number | string | undefined;
+		u2?: number | string | undefined;
+		underlinePosition?: number | string | undefined;
+		underlineThickness?: number | string | undefined;
+		unicode?: number | string | undefined;
+		unicodeBidi?: number | string | undefined;
+		unicodeRange?: number | string | undefined;
+		unitsPerEm?: number | string | undefined;
+		vAlphabetic?: number | string | undefined;
+		values?: string | undefined;
+		vectorEffect?: number | string | undefined;
+		version?: string | undefined;
+		vertAdvY?: number | string | undefined;
+		vertOriginX?: number | string | undefined;
+		vertOriginY?: number | string | undefined;
+		vHanging?: number | string | undefined;
+		vIdeographic?: number | string | undefined;
+		viewBox?: string | undefined;
+		viewTarget?: number | string | undefined;
+		visibility?: number | string | undefined;
+		vMathematical?: number | string | undefined;
+		widths?: number | string | undefined;
+		wordSpacing?: number | string | undefined;
+		writingMode?: number | string | undefined;
+		x1?: number | string | undefined;
+		x2?: number | string | undefined;
+		x?: number | string | undefined;
+		xChannelSelector?: string | undefined;
+		xHeight?: number | string | undefined;
+		xlinkActuate?: string | undefined;
+		xlinkArcrole?: string | undefined;
+		xlinkHref?: string | undefined;
+		xlinkRole?: string | undefined;
+		xlinkShow?: string | undefined;
+		xlinkTitle?: string | undefined;
+		xlinkType?: string | undefined;
+		xmlBase?: string | undefined;
+		xmlLang?: string | undefined;
+		xmlns?: string | undefined;
+		xmlnsXlink?: string | undefined;
+		xmlSpace?: string | undefined;
+		y1?: number | string | undefined;
+		y2?: number | string | undefined;
+		y?: number | string | undefined;
+		yChannelSelector?: string | undefined;
+		z?: number | string | undefined;
+		zoomAndPan?: string | undefined;
+	}
+
+	interface WebViewHTMLAttributes<T> extends HTMLAttributes<T> {
+		allowFullScreen?: boolean | undefined;
+		allowpopups?: boolean | undefined;
+		autosize?: boolean | undefined;
+		blinkfeatures?: string | undefined;
+		disableblinkfeatures?: string | undefined;
+		disableguestresize?: boolean | undefined;
+		disablewebsecurity?: boolean | undefined;
+		guestinstance?: string | undefined;
+		httpreferrer?: string | undefined;
+		nodeintegration?: boolean | undefined;
+		partition?: string | undefined;
+		plugins?: boolean | undefined;
+		preload?: string | undefined;
+		src?: string | undefined;
+		useragent?: string | undefined;
+		webpreferences?: string | undefined;
+	}
+
+	interface ReactSVG {
```

Comment:
Same here, looks like we're duplicating `InstrinsicElements` definitions?

Reply:
@rschristian Yes we can do this.
I just copied them from react type.